k8s/statefulset.go

```diff
@@ -345,7 +345,9 @@ func (m *StatefulSetDesirer) toStatefulSet(lrp *opi.LRP) *appsv1.StatefulSet {
 
 	labels := map[string]string{
 		"guid":                           lrp.GUID,
+		"process_type":                   lrp.ProcessType,
```

Comment:
Would appreciate feedback on these label keys. Just setting it as `type` would be more consistent with `guid` and `version`, but felt imprecise.

Similarly, should `app_guid` just be `app`? I've seen elsewhere (but not here) doing these "relationship" style labels as `<resource-type>: <resource-primary-identifier>`